setup-alpine.sh

```diff
@@ -197,7 +197,7 @@ if needs_emulator "$INPUT_ARCH"; then
 
 	# TODO: Consider replacing it with a simple shell script to speed-up
 	#  the installation.
-	if ! command -V update-binfmts >/dev/null; then
+	if ! command -V update-binfmts 2>/dev/null; then
```

Comment:
Actually, there should be:

```suggestion
	if ! command -V update-binfmts 2>&1 >/dev/null; then
```